### DISCUSSION TITLE
Remote workspace ?

### DISCUSSION THREAD

**Discussion by lukasmrtvy** [eclipse-theia/theia#7729] [General]
Any idea, if is possible to use remote workspace ( http,sftp,webdav,ftp ) ?

**Comment by akosyakov** [comment]
You could host Theia on your remote workspace and access it via the browser?

**Reply by lukasmrtvy** [reply]
What I usually do is that I am syncing remote folder ( mega.nz, gdrive ) between my laptop ( linux ) and my desktop ( windows ), thats ok, but when comes to headless server  ( docker/k8s ), mega.nz nor gdrive does not have clients with bidirectional sync for linux. ( Still, its possible to use rclone for serving cloud backend as http,sftp,webdav,ftp, but its a pain) 

So I am wondering how can all devices can by synced. Its obvious that laptop and desktop will be not online 24/7, but server is ( is possible to use syncthing, but this is not a cloud backend ).

So something like remote workspace will work for me.
